pywinauto/timings.py

```diff
@@ -354,38 +414,9 @@ def WaitUntilPasses(
       except TimeoutError as e:
          print("timed out")
          raise e.
-     
-    """
     
-    start = time.time()
+    """    
+    return always_wait_until_passes(timeout, retry_interval, exceptions)(func)(*args)

```

Comment:
It seems like you moved wait_until* functionality into decorators and link wait_until* on this decorators.
Why not just leave wait_until* functions and cover them by always_wait_until* decorators?